### PR TITLE
notstarted -> notStarted

### PR DESCRIPTION
**Pull request by deyaaeldeen**
The server sends `notStarted` instead of `notstarted` for the health status API:
notstarted

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json [hunk 1]
```diff
@@ -1924,7 +1924,7 @@
         },
         "status": {
           "enum": [
-            "notstarted",
+            "notStarted",
             "running",
             "succeeded",
             "failed",
```
Comment (suhas92) [inline specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: Should `partiallycompleted` be also changes to `partiallyCompeted` ?

**Review by suhas92 — APPROVED** [review]

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json (hunk 1 above)
Comment (deyaaeldeen) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: @suhas92 I do not know, you tell me :)

Comment (abhahn) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: We should loop in @tasharm-0412, he may be able to help answer this question.  I'm looking at the code now but can't find where these values are serialized into JSON so any help in determining what all of the enum values should be would be much appreciated.

Comment (abhahn) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: Ok, actually I think I found it.  Here is how the job status enum is specified on the service side:

```
public enum JobStatus
    {
        Unknown = 0,
        NotStarted,
        Running,
        Cancelling,
        Cancelled,
        PartiallySucceeded,
        Succeeded,
        Failed
    }
```

So my guess is that the serialized versions of all of these values will be camel case, so it is reasonable to assume that "NotStarted" -> "notStarted" and "PartiallySucceeded" -> "partiallySucceeded".

**Review by tjprescott — APPROVED** [review]

**Comment by tjprescott** [comment]
@deyaaeldeen did you want to update `partiallycompleted` or merge just this change?

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json (hunk 1 above)
Comment (deyaaeldeen) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: Thanks I made the change. How about the analysis task state? is it came case too?

**Comment by deyaaeldeen** [comment]
@tjprescott I just updated `partiallycompleted ` as well and i think this is ready for merge for now.

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json (hunk 1 above)
Comment (abhahn) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: Let me see if I can figure out how the orchestrator is representing this.  From the TA side we will be using the same enum, but I need to find out whether or not these values are passed through the orchestrator or if they do any manipulation of the response.

Comment (abhahn) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json:1930]: I see this is already merged, but just to follow up it looks like the enum used on the orchestrator side matches what we have in TA, so this should be fine for both services.

**Pull request merged**